tests/RunAllTests.r

```diff
@@ -1,16 +1,36 @@
-# Uncomment these lines to test local (unpublished) changes to the client
```

Comment:
This test file has been modified to actually run the tests correctly (it didn't actually run the test code before :fearful: ) and then fail if any errors were reported. RUnit is pretty old and no longer maintained; they should probably be ported to a more modern testing framework in the future.